pkg/policy/rule.go

```diff
@@ -327,7 +327,10 @@ func mergeIngress(policyCtx PolicyContext, ctx *SearchContext, fromEndpoints api
 	// restrictions on these endpoints into L7 allow-all so that the
 	// traffic is always allowed, but is also always redirected through the
 	// proxy
-	hostWildcardL7 := option.Config.AlwaysAllowLocalhost()
+	hostWildcardL7 := make([]string, 0, 2)
```

Comment:
why 2? :thinking:

Reply:
ah in this commit makes sense.

Reply:
Yeah strictly speaking this would set to 1 first in the initial refactor patch then 2 in the patch that adds the second entry, I just didn't bother making the two changes since it doesn't make a big difference.